docs/js/wave.js

```diff
@@ -636,6 +640,10 @@ class Wave {
         this.current_stream.id = canvas_id;
         this.current_stream.options = options;
 
+
+        var AudioContext = window.AudioContext          // Default
```

Comment:
@CorfitzMe Could we declare it only once globally instead? (Outside from the class might be the right place)

Reply:
Good idea on this matter....